yarpl/test/FlowableTest.cpp

```diff
@@ -180,6 +180,14 @@ TEST(FlowableTest, SimpleTake) {
   EXPECT_EQ(std::size_t{0}, Refcounted::objects());
 }
 
+TEST(FlowableTest, IgnoreElements) {
+  EXPECT_EQ(std::size_t{0}, Refcounted::objects());
```

Comment:
It'd be nice to have a test for verifying `Flowables::error(...)->ignoreElements()` propagates an error through successfully, but this operator is simple enough that I think it's fine.